tldw_Server_API/app/core/Web_Scraping/outbound_policy.py

```
@@ -0,0 +1,205 @@
+
+    if not respect_robots:
+        return _decision(True, mode=mode, reason="robots_skipped", stage=stage, source=source)
+
+    robots_filter = robots_filter or RobotsFilter(user_agent=user_agent or "*")
+    try:
+        robots_result = await robots_filter.check(
+            url,
+            skip_egress_check=True,
+            fail_open=(mode != "strict"),
+        )
+    except Exception as exc:  # pragma: no cover - explicit behavior covered by monkeypatched tests
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)